src/create_venv.sh

```diff
@@ -54,6 +54,9 @@ yb::verbose_log "Using ${yb_python_interpreter} (${yb_python_version_actual})"
 # Internal functions used in this module.  These shouldn't be called directly outside this module.
 # -------------------------------------------------------------------------------------------------
 
+# Needed for deterministic sort order
+LC_COLLATE=C.UTF8
```

Comment:
Don't we need to export this? Also instead of setting this in the global namespace, why not set it for the `sort` command only, e.g. `tmp=$( export LC_COLLATE=C.UTF8; sort -u <<<...`?

Reply:
This works in my testing without the export.  My thinking for putting it at the top like this is so we don't have to remember to add explicit LC_COLLATE to every call to sort since we call sort in a few places.

Reply:
Modified it to only set LC_COLLATE for calls to sort.